src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -233,10 +233,14 @@ public boolean verifyTitle(String title) {
     return true;
   }
 
-  public void verifyURLcontains(String givenString) {
+  public void verifyURLcontains(String exptectedString) {
     String currentURL = driver.getCurrentUrl();
-    Assertion.assertStringContains(currentURL.toLowerCase(), givenString.toLowerCase());
-    PageObjectLogging.log("verifyURLcontains", "current url is the same as expetced url", true);
+    if (Assertion.assertStringContains(currentURL.toLowerCase(), exptectedString.toLowerCase())) {
+      PageObjectLogging.log("verifyURLcontains", "current url contains " + exptectedString, true);
+    } else {
+      PageObjectLogging
+          .log("verifyURLcontains", "current url doesn't contain " + exptectedString, true);
```

Comment:
I'll rename the method to isStringInURL

for the green background issue - it is enought to change the parameter "true" into "false" in the logging method. Thanks for the remark !